app/services/RunningSum.coffee

```diff
@@ -3,15 +3,17 @@ $ = require 'jquery'
 class RunningSum
 
   @display: (solution, value) ->
-    if solution is ''
-      expression = ''
-    else if isNaN value
-      expression = 'Invalid Expression'
-    else if @isCompleteExpression solution
-      expression = (@addParens solution) + '=' + value
-    else
-      expression = solution
-    $('#running-sum').html(@format expression)
+    unless $('#running-sum').html() is 'Try to get all the tiles off the board!'
+      if solution is '' or solution is 'Try to get all the tiles off the board!' or
+          solution is 'Solution must include an operator'
```

Comment:
Save these 'magic strings' as class variables, or create some kind of master lookup if they're needed in other files so that we can edit them easily.